loops/until-loops.sh

```diff
@@ -3,8 +3,45 @@
 # Tutorial: brief description of tutorial content
```

Comment:
Update the tutorial description here to say this is a bash tutorial about until-loops